quesma/quesma/config/config.go

```diff
@@ -44,9 +44,10 @@ type QuesmaConfiguration struct {
 	DisableAuth                bool
 	AutodiscoveryEnabled       bool
 
-	EnableIngest      bool // this is computed from the configuration 2.0
-	CreateCommonTable bool
-
+	EnableIngest                 bool // this is computed from the configuration 2.0
+	CreateCommonTable            bool
+	CreateCommonTableForWildcard bool // the meaning of this is to create a common table for wildcard indexes
```

Comment:
```suggestion
	UseCommonTableForWildcard bool // the meaning of this is to create a common table for wildcard indexes
```

Such naming maybe would make things clearer?

Reply:
I agree, but should not we change `CreateCommonTable` as well to be consistent?

Reply:
done